static/sass/main.scss

```diff
@@ -1,5 +1,57 @@
-@import 'settings_colors';
-@import 'vanilla-framework/scss/build';
+@import "settings_colors";
+@import "vanilla-framework/scss/build";
 
 // import cookie policy
 @import "@canonical/cookie-policy/build/css/cookie-policy";
+
+//import additional styles
+@import "pattern_strip.scss";
+@import "pattern_github_buttons";
+
+@include multipass-p-strip;
+@include multipass-github-buttons;
+
+//Custom styles
+
+.p-tab {
+  color: $color-mid-dark;
+
+  &:focus,
+  &:hover,
+  &.active {
+    background-color: $color-light;
+    text-decoration: none;
+  }
+}
+
+.p-navigation__banner {
```

Comment:
It would be tidier to move these styles into two pattern files "pattern_tabs.scss" and "pattern_navigation.scss".